src/Form/LockerRequestForm.php

```diff
@@ -219,7 +219,7 @@ function arborcat_mail($key, $email_to, $patron, $code, $holds) {
 			foreach($holds as $title){
 				$email_message = $email_message.$title['Title']."\r\n";
 			}
-			$mailManager = \Drupal::service('plugin.manager.mail');
+			$mail_manager = \Drupal::service('plugin.manager.mail');
```

Comment:
Looks like this is the only reference to this variable. Does this line even need to be here?